agent/firewall.go

```diff
@@ -407,12 +407,14 @@ func provisionFirewallRules(netif NetIf, agent *Agent) error {
 	if err != nil {
 		return err
 	}
-	for chain := range missingChains {
+	for _, chain := range missingChains {
 		if err := fw.CreateRules(chain); err != nil {
 			return err
 		}
-		if err := fw.CreateU32Rules(chain); err != nil {
-			return err
+		if chain == forwardChainIndex {
```

Comment:
Yup. Or we could refactor it a bit to add the u32 rules here: https://github.com/romana/core/blob/843d5fb64d52cd378ecb9b1d04240f04f4e7a4d5/agent/firewall.go#L125